code/__DEFINES/food.dm

```diff
@@ -18,6 +18,7 @@
 #define ORANGES (1<<17)
 #define BUGS (1<<18)
 #define GORE (1<<19)
+#define BLOODY (1<<20) // SKYRAT EDIT - Hemophage Food
```

Comment:
i feel similar here but this one might be too engrained

Reply:
Yeah it sadly is a bit too engrained. I'll deal with the conflict when it happens, and if it's no longer possible to do it this way I'll find another way.